docs/aci-compose-features.md

```diff
@@ -0,0 +1,118 @@
+| service.deploy                 | ✓  |                                                              |
+| service.deploy.endpoint_mode   | x  |                                                              |
+| service.deploy.mode            | x  |                                                              |
+| service.deploy.replicas        | x  |  Only one replica is started for each service with the docker-ACI integration. | 
+| service.deploy.placement       | x  |  
+| service.deploy.update_config   | x  |  
+| service.deploy.resources       | ✓  |  Restriction: ACI Resource limits cannot be greater than the sum of resource requests for all containers in the container group. This will allow one containers in the same container group to compete with resources. 
+| service.deploy.restart_policy  | ✓  |  One of: any - none - on-failure. Restriction: All services must have the ame restart policy. The entire ACI container group will be restarted if needed. | 
+| service.deploy.labels          | x  |  ACI does not have container-level labels. In addition, Azure tag are too restrictive to support Docker container labels.| 
+| service.devices                | x  | 
+| service.depends_on             | x  | 
+| service.dns                    | x  | 
+| service.dns_search             | x  | 
+| service.domainname             | ✓  |  Mapped to ACI DNSLabelName. Restriction: all services must specify the same domainname, if specified.
+| service.tmpfs                  | x  |  
+| service.entrypoint             | x  |  ACI allows to override Command, but not Entrypoint. 
+| service.env_file               | ✓  |  
+| service.environment            | ✓  |  
+| service.expose                 | x  |   
+| service.extends                | x  |  
+| service.external_links         | x  |  
+| service.extra_hosts            | x  |  
+| service.group_add              | x  |  
+| service.healthcheck            | x  |  
+| service.hostname               | x  |  
+| service.image                  | ✓  |  Private images will be accessible if the user is logged in against the corresponding registry at deploy time. Users will be automatically logged in to Azure Container Registry from Azure login if possible.
+| service.isolation              | x  |  
+| service.labels                 | x  |  ACI does not have container-level labels. In addition, Azure tag are too restrictive to support Docker container labels.  |
+| service.links                  | x  |  
+| service.logging                | x  |  
+| service.network_mode           | x  |  
+| service.networks               | x  |  Communication between services is supported by defining mapping for each service in `/etc/hosts` in the container group. Each service can resolve names for other services and the resulting network calls will be redirected to `localhost`.
+| service.pid                    | x  |  
+| service.ports                  | ✓  |  
```

Comment:
It is probably worth mentioning that only symmetric port mapping is supported

Reply:
I see that you talk about it below, but such important limitation deserves a comment here as well IMO